packages/core-plugin-api/src/routing/useRouteContext.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2020 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useVersionedContext } from '@backstage/version-bridge';
+import { RouteResolver } from './useRouteRef';
+
+export function useRouteContext(): RouteResolver | undefined {
```

Comment:
I'm curious how exporting this helps? `useRouteRef` is already a pretty thin wrapper around this that just calls `resolve` with the existing location

Reply:
Yeah, this was so that one can iterate the locations to view the whole route table as in #10260. I didn't find any other way to expose that, but I could surely have missed something.